Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcCopyClient.cs

```diff
@@ -60,6 +94,7 @@ public async Task<BoolResult> ShutdownAsync(Context context)
             }
 
             ShutdownCompleted = true;
+            _clientPool.TryRemove((_host, _grpcPort), out var copyClient);

```

Comment:
Should we assert that TryRemove returns true?

Reply:
Removing _clientPool

---
In reply to: [271135924](https://github.com/Microsoft/BuildXL/pull/66#discussion_r271135924) [](ancestors = 271135924)